src/rules/__tests__/createEvaluateConditions.test.js

```diff
@@ -37,114 +37,99 @@ describe('createEvaluateConditions returns a function that when called', functio
+      // Not using a real promise to ensure that the logic detects
+      // objects that are promise-like.
+      return {
+        then: function () {}
+      };
+    };
+    var isConditionMet = jasmine.createSpy();
+    var logConditionNotMetSpy = jasmine.createSpy();
+    var logConditionErrorSpy = jasmine.createSpy();
+    var result = createEvaluateConditions(
+      executeDelegateModule,
+      isConditionMet,
+      logConditionNotMetSpy,
+      logConditionErrorSpy
+    )(rule);
 
-    expect(
-      createEvaluateConditions(
-        executeDelegateModule,
-        isConditionMet,
-        logConditionNotMet,
-        logConditionError
-      )(rule)
-    ).toBeFalse();
+    expect(result).toBeFalse();
+    expect(isConditionMet).not.toHaveBeenCalled();
+    expect(logConditionNotMetSpy).not.toHaveBeenCalled();
+    expect(logConditionErrorSpy).toHaveBeenCalledWith(
+      rule.conditions[0],
+      rule,
+      new Error(
+        'Rule component sequencing must be enabled on the property ' +
+          'for this condition to function properly.'
+      )
+    );
   });
 
-  it('calls logConditionNotMet when conditions are not met', function () {
+  it('handles a condition not being met', function () {
     var executeDelegateModule = function (condition) {
       return condition.modulePath === 'condition1';
     };
     var isConditionMet = function () {
       return false;
     };
-    var logConditionNotMetSpy = jasmine.createSpy('logConditionNotMet');
-    var logConditionError = emptyFn;
-    var condition = {
-      modulePath: 'condition1'
-    };
-    var rule = {
-      conditions: [condition]
-    };
 
-    createEvaluateConditions(
+    var logConditionNotMetSpy = jasmine.createSpy();
+    var logConditionErrorSpy = jasmine.createSpy();
+    var result = createEvaluateConditions(
       executeDelegateModule,
       isConditionMet,
       logConditionNotMetSpy,
-      logConditionError
+      logConditionErrorSpy
     )(rule);
 
-    expect(logConditionNotMetSpy).toHaveBeenCalledWith(condition, rule);
-  });
-
-  it('returns false when a condition throws an error', function () {
-    var executeDelegateModule = function () {
-      throw new Error('some error');
-    };
-    var isConditionMet = emptyFn;
-    var logConditionNotMet = emptyFn;
-    var logConditionError = emptyFn;
-
-    expect(
-      createEvaluateConditions(
-        executeDelegateModule,
-        isConditionMet,
-        logConditionNotMet,
-        logConditionError
-      )({
-        conditions: [
-          {
-            modulePath: 'condition1'
-          }
-        ]
-      })
-    ).toBeFalse();
+    expect(result).toBeFalse();
+    expect(logConditionNotMetSpy).toHaveBeenCalledWith(
+      rule.conditions[0],
+      rule
+    );
+    expect(logConditionErrorSpy).not.toHaveBeenCalled();
   });
 
-  it('calls logConditionError when a condition throws an error', function () {
+  it('handles a condition throwing an error', function () {
```

Comment:
This test case is the result of merging two test cases that were duplicating the same scenario but that had two different assertions.